posthog/src/main/java/com/posthog/internal/PostHogRemoteConfig.kt

```diff
@@ -381,39 +381,17 @@ internal class PostHogRemoteConfig(
         return decideResponse
     }
 
-    fun isFeatureEnabled(
-        key: String,
-        defaultValue: Boolean,
-    ): Boolean {
+    private fun loadFeatureFlagsFromCacheIfNeeded() {
         if (!isFeatureFlagsLoaded) {
             loadFeatureFlagsFromCache()
         }
-        val value: Any?
-
-        synchronized(featureFlagsLock) {
-            value = featureFlags?.get(key)
-        }
-
-        return if (value != null) {
-            if (value is Boolean) {
-                value
-            } else {
-                // if its multivariant flag, its enabled by default
-                true
-            }
-        } else {
-            defaultValue
-        }
     }
 
     private fun readFeatureFlag(
         key: String,
         defaultValue: Any?,
         flags: Map<String, Any?>?,
     ): Any? {
-        if (!isFeatureFlagsLoaded) {
```

Comment:
I call `loadFeatureFlagsFromCacheIfNeeded` before calling `readFeatureFlag` otherwise `readFeatureFlag` will always try to read the `flags` before its loaded from cache, so always null